ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sha256

```diff
@@ -1 +1 @@
-b79475730d1275f07bcbd35d9133ae347c5af9abf03e5dab9a24b219ce3c958b
+e71124d50f7331554432d23e921088ea20e06055cccfdc18efeb4fa76761ebea
```

Comment:
I confirm we do not promise data continuity for H2 for index DB.
The V1__Append_only_schema.sql file is edited in place since introduction of append only.
I would claim we shouldn't have H2, that would solve the H2 issue with checksums (otherwise I agree, but don't think we need this change, since we want to drop H2).